src/backend/aspen/test_infra/models/sequences.py

```diff
@@ -35,3 +36,31 @@ def sequencing_read_factory(
             public_identifier=public_identifier,
         )
     return sequencing_reads
+
+
+def uploaded_pathogen_genome_factory(
+    sample,
+    sequence=">test1 NTCGGCG",
```

Comment:
I think the nucleotide data should be after a NL.

```suggestion
    sequence=">test1\nNTCGGCG",
```